api/v1alpha3/conversion.go

```diff
@@ -113,6 +114,19 @@ func Convert_v1alpha3_OpenStackClusterSpec_To_v1alpha4_OpenStackClusterSpec(in *
 	return autoConvert_v1alpha3_OpenStackClusterSpec_To_v1alpha4_OpenStackClusterSpec(in, out, s)
 }
 
+// Convert_v1alpha4_OpenStackClusterSpec_To_v1alpha3_OpenStackClusterSpec has to be added by us because we have to
+// convert the Type of CloudsSecret from SecretReference to string.
+func Convert_v1alpha4_OpenStackClusterSpec_To_v1alpha3_OpenStackClusterSpec(in *v1alpha4.OpenStackClusterSpec, out *OpenStackClusterSpec, s conversion.Scope) error {
+	if err := autoConvert_v1alpha4_OpenStackClusterSpec_To_v1alpha3_OpenStackClusterSpec(in, out, s); err != nil {
+		return err
+	}
+	out.CloudsSecret = new(corev1.SecretReference)
+	if in.CloudsSecret != nil {
+		out.CloudsSecret.Name = *in.CloudsSecret
+	}
+	return nil
+}
+
 // Convert_v1alpha3_OpenStackMachineSpec_To_v1alpha4_OpenStackMachineSpec is an autogenerated conversion function.
 // v1alpha4 drops the field .UserDataSecret which is why we reuqire to define the function here.
 func Convert_v1alpha3_OpenStackMachineSpec_To_v1alpha4_OpenStackMachineSpec(in *OpenStackMachineSpec, out *v1alpha4.OpenStackMachineSpec, s conversion.Scope) error {
```

Comment:
Could you change the doc in line 136 to `Convert_v1alpha4_Network_To_v1alpha3_Network`.

Reply:
Sure, done 👍🏻